File: plugins/basis/definition.lua
return {
  protocol = 1,
  version = "0.0.1",
  dependencies = {},
  setup = function(p)
    -- Must have block type index 0
    p.define_block_type("unknown", {
      client_hints = { color = "#000" }
    })

    -- Must have block type index 1
    p.define_block_type("empty", {
      client_hints = { color = "#fff" }
    })
  end
}

File: plugins/basis/definition.js
module.exports = {
  protocol: 1,
  version: "1.0.0",
  dependencies: [],
  setup: (p) => {
    // Must have block type index 0
    p.defBlockType("unknown", { clientHints: { color: "#000" } });

    // Must have block type index 1
    p.defBlockType("empty", { clientHints: { color: "#fff" } });
  },
};
